imap_processing/cdf/config/imap_swapi_variable_attrs.yaml

```diff
@@ -0,0 +1,104 @@
+# <=== Coordinates ===>
+energy:
+  CATDESC: Energy step id in lookup table
+  FIELDNAM: Energy Step
+  FILLVAL: -9223370000000000000
+  FORMAT: I2
+  LABLAXIS: Energy Step
+  SCALE_TYP: linear
+  UNITS: " "
+  VALIDMAX: 71
+  VALIDMIN: 0
+  VAR_TYPE: support_data
+
+# <=== Label Attributes ===>
+energy_label:
```

Comment:
Doesn't this need to inherit or include required variables?

Reply:
This is of metadata type and only used for LABL_PTR for 2D and higher dimension. I believe it only requires these four fields.